manifests/z/Zoom/Zoom/5.15.11.21032/Zoom.Zoom.installer.yaml

```diff
@@ -0,0 +1,101 @@
+# Created with Komac v1.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: Zoom.Zoom
+PackageVersion: 5.15.11.21032
+InstallerLocale: en-US
+UpgradeBehavior: install
+Protocols:
+- zoom
+- zoommtg
+FileExtensions:
+- zoommtg
+- zoomphonecall
+ReleaseDate: 2023-08-28
+Installers:
+- Architecture: x64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://cdn.zoom.us/prod/5.15.11.21032/x64/ZoomInstallerFull.exe
+  InstallerSha256: 882E8602F81881D4158DD5897825DEA7A1D308973ABCF29411F299F16E8DE0EA
+  InstallModes:
+  - interactive
+  - silent
+  InstallerSwitches:
+    Silent: /silent
+    SilentWithProgress: /silent
+  ProductCode: ZoomUMX
+  AppsAndFeaturesEntries:
+  - DisplayVersion: 5.15.11 (21032)
+    ProductCode: ZoomUMX
+- Architecture: arm64
+  InstallerType: exe
+  Scope: user
+  InstallerUrl: https://cdn.zoom.us/prod/5.15.11.21032/arm64/ZoomInstallerFull.exe
+  InstallerSha256: F4CBBEFBDC4B5615315E8F7641E106C601A8DBF345B67A96FB0AB09D61876C2F
+  InstallModes:
+  - interactive
+  - silent
+  InstallerSwitches:
+    Silent: /silent
+    SilentWithProgress: /silent
+  ProductCode: ZoomUMX
+  AppsAndFeaturesEntries:
+  - DisplayVersion: 5.15.11 (21032)
+    ProductCode: ZoomUMX
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: msi
+  Scope: user
+  InstallerUrl: https://cdn.zoom.us/prod/5.15.11.21032/ZoomInstallerFull.msi
+  InstallerSha256: 9819BEB511A70D61E3EFFB1E0BA38210447E7FC0C1716DA54BEC127D89EDC57D
+  InstallModes:
+  - interactive
+  - silent
+  InstallerSwitches:

```

Comment:
Looks like these might be the issue
![IMG_1186](https://github.com/microsoft/winget-pkgs/assets/12611259/071a4277-f162-4ab6-bf8f-ad228bdb0707)